spectrocloud/provider.go

```diff
@@ -37,6 +37,11 @@ func New(_ string) func() *schema.Provider {
 					Sensitive:   true,
 					DefaultFunc: schema.EnvDefaultFunc("SPECTROCLOUD_APIKEY", nil),
 				},
+				"transport_debug": &schema.Schema{
```

Comment:
We can rename it as `trace`
Generally, `trace` means a higher level of `debug` which prints all API calls.